.travis.yml

```diff
@@ -28,6 +28,7 @@ services:
   - mysql
   - memcached
   - elasticsearch
+  - redis
```

Comment:
Celery now runs eager tasks through their broker + result backend accordingly so we need redis on travis now.

This doesn't mean that "always eager" is encouraged now, it's still discouraged but we are using it so heavily that I didn't want to change that just now.